src/components/30-organisms/modal/index.js

```diff
@@ -111,7 +111,7 @@ class AXAModal extends InlineStyles {
     this.inlineStyles('resetHeadingCss');
 
     // add eventListener to close modal when pressing outside the modal
-    document.body.addEventListener('click', ev => {
+    document.body.addEventListener('mousedown', ev => {
```

Comment:
Since it is unusual to not use `click` here, a comment is mandatory explaining what problem is solved by switching to `mousedown`!